spec/index.html

```diff
@@ -52,13 +53,16 @@
   </script>
 
   <style>
+    body { color-scheme: light; }
```

Comment:
This prevents darkmode to work when JS is not enabled but not sure if it's a big deal.

Reply:
Agreed; with the added suggestion to remove those two lines to avoid required further changes (the use of `color-scheme` here should only affect SVGs).